src/library_gl.js

```diff
@@ -350,7 +350,7 @@ var LibraryGL = {
 #if GL_ASSERTIONS
         GL.validateVertexAttribPointer(cb.size, cb.type, cb.stride, 0);
 #endif
-        GLctx.vertexAttribPointer(i, cb.size, cb.type, cb.normalized, cb.stride, 0);
+        cb.vertexAttribPointer.call(GLctx, i, cb.size, cb.type, cb.normalized, cb.stride, 0);
```

Comment:
Perhaps we should rename the function here. As it may refer to either `vertexAttribPointer` or `vertexAttribIPointer`, a more generic name that implies it could be either might be better. Perhaps something generic like `setup`, or maybe specific like `vertexAttribXPointer` (X implying it might be one of multiple values)? A benefit of a more unique name is that a search for that name in this file would find just the specific things that this property is used for.